src/content/docs/opentelemetry/nrdot/nrdot-intro.mdx

```diff
@@ -0,0 +1,20 @@
+---
+title: New Relic Distribution of OpenTelemetry (NRDOT)
+tags:
+  - OpenTelemetry
+  - NRDOT
+metaDescription: New Relic Supported Distribution of OpenTelemetry
+freshnessValidatedDate: 2024-05-14
+---
+
+The New Relic Distribution of OpenTelemetry (NRDOT) is a New Relic-supported distribution of the OpenTelemetry project.
```

Comment:
I think this sentence is making too strong a claim right now. Maybe this is the aspirational goal, but not the case right now.

Maybe something like, "New Relic Distribution of OpenTelemetry (NRODT) refers to the collection of New Relic-supported distributions of OpenTelemetry project components.